aleph-client/src/lib.rs

```diff
@@ -105,21 +109,40 @@ pub fn create_custom_connection<Client: FromStr + RpcClient>(
     }
 }
 
-pub fn send_xt(connection: &Connection, xt: String, xt_name: &'static str, tx_status: XtStatus) {
-    let block_hash = connection
-        .send_extrinsic(xt, tx_status)
-        .expect("Could not send extrinsic")
-        .expect("Could not get tx hash");
-    let block_number = connection
-        .get_header::<Header>(Some(block_hash))
-        .expect("Could not fetch header")
-        .expect("Block exists; qed")
-        .number;
-    info!(
-        target: "aleph-client",
-        "Transaction {} was included in block {}.",
-        xt_name, block_number
-    );
+/// `panic`able utility wrapper for `try_send_xt`.
+pub fn send_xt<T: Encode>(
+    connection: &Connection,
+    xt: UncheckedExtrinsicV4<T>,
+    xt_name: Option<&'static str>,
+    xt_status: XtStatus,
+) -> Option<H256> {
```

Comment:
If we're changing API in a non-backwards compatible way we should bump minor version of the client.

Reply:
true, I bumped patch part before and forgot to adjust version to the newest changes; fixed